challangeNetflix/MovieDetails/MovieDetailsPresenter.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  MovieDetailsPresenter.swift
+//  challangeNetflix
+//
+//  Created by Larissa Diniz  on 12/05/20.
+//  Copyright © 2020 Larissa Diniz . All rights reserved.
+//
+
+import Foundation
+
+protocol MovieDetailsPresentable: class {
+    var view: MovieViewDetails? { get }
+    func showItem(item: Movie?)
+    func callToggleFavButton(isFavorite: Bool)
+}
+
+final class MovieDetailsPresenter: MovieDetailsPresentable {
+    
+    internal weak var view: MovieViewDetails?
+    
+    init(view: MovieViewDetails) {
+        self.view = view
+    }
+    
+    func callToggleFavButton(isFavorite: Bool) {
+        view?.toggleFavButton(isFavorite: isFavorite)
+    }
+    
+    func showItem(item: Movie?) {
```

Comment:
Lembra da dica, lê a função incluindo o parametro: show item item?
```suggestion
func show(item: Movie?) {
```
ou
```suggestion
func show(movie: Movie?) {
```